tests/bootstrap.php

```diff
@@ -7,6 +7,14 @@
     throw new RuntimeException('Install dependencies to run test suite.');
 }
 
+$files = array_filter(array(
+    __DIR__.'/../vendor/symfony/symfony/src/Symfony/Bridge/PhpUnit/bootstrap.php',
+    __DIR__.'/../vendor/symfony/phpunit-bridge/bootstrap.php',
+), 'file_exists');
+if ($files) {
+    require_once current($files);
+}
```

Comment:
as this library will never install the `symfony/symfony` package during tests, we know that the bootstrap file is available in ``../vendor/symfony/phpunit-bridge/bootstrap.php``.